### DISCUSSION TITLE
Write a Tutorial how to use avalonia with linux framebuffer

### DISCUSSION THREAD

**Discussion by furesoft** [AvaloniaUI/Avalonia#5521] [Ideas]
Write a Tutorial how to use avalonia with linux framebuffer

**Comment by JCH2k** [comment]
This is basically what you're asking for:
https://docs.avaloniaui.net/docs/guides/platforms/rpi/running-on-raspbian-lite-via-drm

Just replace `StartLinuxDrm()` with `StartLinuxFbDev()` and you're good to go!
However depending on your HW you shoud consider using DRM for better performance.
